cmd/create/ocmrole/cmd.go

```diff
@@ -123,10 +123,10 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	// Determine if managed policies are enabled
+	// Determine if Classic ROSA managed policies are enabled
 	isManagedSet := cmd.Flags().Changed("managed-policies") || cmd.Flags().Changed("mp")
 	if isManagedSet && env == ocm.Production {
```

Comment:
Shouldn't this check be removed?

Reply:
Initially, the requirement for Classic ROSA managed policies was for the OCM role as well.
But, as it seems, the OCM role will have unmanaged policies for both topologies.
Because some permissions in the OCM role can't be included in a managed policy.
The fix for this command will be taken care of in the Classic ROSA managed policies epic, and it's currently blocked.